lib/tasks/staging_seeds.rake

```diff
@@ -0,0 +1,49 @@
+# frozen_string_literal: true
+
+namespace :comfy do
+    desc "Import CMS Seed data from staging"
+
+    task :seed_import => :environment do |_t|
+      require 'net/ssh'
+      require 'net/scp'
+      
+      def check_timestamp(file, session, local_file, remote_file)     
+        newer = Time.at(file.attributes.mtime) >= File.stat(local_file).mtime 
```

Comment:
I would rename this variable `is_newer` so that's immediately understandable it's a boolean flag